pkg/sachecker/main.go

```diff
@@ -0,0 +1,93 @@
+package sachecker
+
+import (
+	"fmt"
+	"os"
+
+	w "github.com/sfowl/pod-checker/pkg/cmdwrapper"
+	h "github.com/sfowl/pod-checker/pkg/helpers"
+	log "github.com/sirupsen/logrus"
+)
+
+type SAChecker struct {
+	namespace          string
+	serviceAccountName string
+	group              string
+}
+
+func NewSAChecker(namespace string, serviceAccountName string, group string) SAChecker {
+	c := SAChecker{}
+	c.namespace = namespace
+	c.serviceAccountName = serviceAccountName
+	c.group = group
+
+	return c
+}
+
+func (c *SAChecker) Run() {
+	c.runRBACTool()
+	c.getSATokens()
+}
+
+func (c *SAChecker) runRBACTool() {
+	rbacToolArgs := []string{
+		"rbac-tool",
+		"policy-rules",
+		"--output", "json",
+		"-e", fmt.Sprintf("^(system:serviceaccounts:%s|%s)$", c.namespace, c.serviceAccountName),
+	}
+
+	log.Infof("Starting rbac-tool for service account %s in namespace: %s", c.serviceAccountName, c.namespace)
+
+	c.runOC(rbacToolArgs, "rbac")
+
+	log.Infof("Finished execution of rbac-tool for service account %s", c.serviceAccountName)
+}
+
+func (c *SAChecker) getSATokens() {
+	getSAParams := []string{
```

Comment:
It would be nice if there was a way to call this plugin from the client library rather than needing to exec `oc`, might be more work than it's worth though